QUESTIONS.md

```diff
@@ -3,3 +3,20 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. How can we better troubleshoot things in node_modules when it won't load or initialize installed packages?
+
+2. What are some ways I can manage burnout and paralysis analysis?
+
+3. Advice on how to maintain continuity of thought when working on a long-term project?
+
+4. Can we still keep in touch after the cohort ends regarding coding questions/problems?
```

Comment:
I know that Hollis suggested we set up a Facebook group for us all to stay connected. I think LinkedIn will be helpful to stay connected with Stack Education.